### PR TITLE
fix error in some previous versions of Chrome

### PR DESCRIPTION
**Pull request by jingt06**
- add blacklistedTypes
- property is blacklisted if type is MimeType

### CODE REVIEW

File: src/ObjectGraph.es6.js [hunk 1]
```diff
@@ -74,6 +74,10 @@ ObjectGraph.prototype.blacklistedObjects = [
 }.bind(typeof window === 'undefined' ? global : window)).filter(
   value => value !== undefined
 );
+ObjectGraph.prototype.blacklistedTypes = [
+  'MimeType'   // causing error in some previous versions of Chrome
```
Comment (mdittmer) [inline src/ObjectGraph.es6.js]: nit: whenever possible, phrase comment as a sentence (start with capital letter, end with period). Also, please be more specific; e.g.,

"This type initiates a process of continually extending the object graphs in Chrome <= [[largest version number with issue]]."

**Review by mdittmer — CHANGES_REQUESTED** [review]
This is almost ready. Just one nit.

Before mergine, please:

(1) Create an issue detailing the problem you're seeing in old version of Chrome.
(2) Mention the issue # in a comment on this PR.

Don't mark the issue as fixed yet. We should investigate *exactly* what's going on with Mime types and try to capture that data exactly once, rather than excluding it.

**Comment by jingt06** [comment]
Issue #4

File: src/ObjectGraph.es6.js [hunk 2]
```diff
@@ -74,6 +74,11 @@ ObjectGraph.prototype.blacklistedObjects = [
 }.bind(typeof window === 'undefined' ? global : window)).filter(
   value => value !== undefined
 );
+ObjectGraph.prototype.blacklistedTypes = [
+  'MimeType'   // MimeType Object initiates a process of continually
+               // extending the object graphs in Chrome
```
Comment (mdittmer) [inline src/ObjectGraph.es6.js]: What's the largest version number where this issue occurs? I think we should include it in the comment.

nit: Still missing period at end of comment.

**Review by mdittmer — CHANGES_REQUESTED** [review]
Just some more comment clarification stuff.

**Review by mdittmer — APPROVED** [review]
LGTM!

**Pull request merged**